### ISSUE TITLE
response with HTML content being lost

### ISSUE THREAD

**Issue by chluther** [a85/POSTMan-Chrome-Extension#1890]
In recent update (Chrome 4.1.0) all HTML encoded in the response from our API endpoint is being pruned.  This did not happen in an earlier release.  

Sending the request via CURL the result returns (edited for clarity)

```
mails.newVerification.failure.statusMessage,Failed to send email,Failed to send email,Failed to send email,Failed to send email
mails.newVerification.htmlBody,"<!DOCTYPE html PUBLIC ""-//W3C//DTD XHTML 1.0 Transitional//EN"" ""http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"">\n<html xmlns=""http://www.w3.org/1999/xhtml"">\n<head>\n<meta name=""viewport"" content=""width=device-width"" />\n<meta http-equiv=""Content-Type"" content=""text/html; charset=UTF-8"" />\n
...</html>\n"
mails.newVerification.subject,Thank you for registering.,Thank you for registering.,Thank you for registering.
mails.newVerification.success.statusMessage,Successfully sent email.,Successfully sent email.,Successfully sent email.,Successfully sent email.
mails.newVerification.textBody,"Thank You for registering. \n\nCongratulations
 ... \n"
```

The result displayed by the same API call via the Postman Client.  Note that the **mails.newVerification.htmlBody** element is missing.

![p](https://www.evernote.com/shard/s17/sh/cef4c764-35d7-4980-9bdc-680b911a6dfe/f8450cec3c6029cc/res/b23547c0-a863-4c51-b01a-97efb810a8a8/skitch.png)

https://www.evernote.com/l/ABHO9MdkNddJgJvcaAuRGm3--EUM7DxgKcw

**Comment by abhijitkane** [comment]
@chluther Can you include the response headers (The Headers link next to Body and Cookie)? Is the response shown correctly in the pretty view?

Also, the DevTools window (http://blog.getpostman.com/2015/06/13/debugging-postman-requests/) might be useful here. The Network tab will show you the response payload that Postman received. Is that part correct?
If so, you can check for any errors in the DevTools' console tab.

**Comment by chluther** [comment]
@abhijitkane - Thank you for the quick response.  The response was displaying similarly in the pretty view.  I followed your diagnosis recommendations and for whatever reason the response started coming back correctly.  I don't know what if anything I changed.  It is hard to know if the issue was with the API or the client. But it worked from CLI, and since there had been a recent upgrade in the client I just assumed it was the client.  Thanks anyway.
